erroranalysis/setup.py

```diff
@@ -15,6 +15,10 @@
 with open('requirements.txt') as f:
     install_requires = [line.strip() for line in f]
 
+extras_require = {
```

Comment:
do you need to add this to the error analysis CI (github action) to run the tests? I'm guessing it's skipping the test now?

Reply:
Great catch, thank you!